docs/data-ai/ai/alert.md

```diff
@@ -7,51 +7,63 @@ type: "docs"
 description: "Use triggers to send email notifications when inferences are made."
 ---
 
-At this point, you should have already set up and tested [computer vision functionality](/data-ai/ai/run-inference/).
-On this page, you'll learn how to use triggers to send alerts in the form of email notifications or webhook requests when certain detections or classifications are made.
+Triggers can send alerts in the form of email notifications or webhook requests when a service makes a specific detection or classification.
 
-You will build a system that can monitor camera feeds and detect situations that require review.
-In other words, this system performs anomaly detection.
-Whenever the system detects an anomaly, it will send an email notification.
-
-First, you'll set up data capture and sync to record images with the anomaly and upload them to the cloud.
-Next, you'll configure a trigger to send email notifications or webhook requests when the anomaly is detected.
+For example, a trigger could alert you when a camera feed detects an anomaly.
 
 ### Prerequisites
 
-{{% expand "A running machine connected to the Viam app. Click to see instructions." %}}
+{{% expand "A running machine connected to the Viam app." %}}
 
 {{% snippet "setup-both.md" %}}
 
 {{% /expand%}}
 
-{{< expand "A configured camera and vision service. Click to see instructions." >}}
+{{< expand "A configured camera and vision service." >}}
 
 Follow the instructions to [configure a camera](/operate/reference/components/camera/) and [run inference](/data-ai/ai/run-inference/).
 
 {{< /expand >}}
 
 ## Configure a filtered camera
```

Comment:
We may want to add a caveat here that other modules might also allow you to do this or that you could build a module to do this. In theory it's not limited to images either. You can address this in future but something to keep in mind